syntax = "proto3";

package sbai_geometry_protos;

// This represents an orientation in free space in quaternion form.

message Quaternion {
    double x = 1;
    double y = 2;
    double z = 3;
    double w = 4;
};
